services/realtimeclock.md

```diff
@@ -0,0 +1,33 @@
+# Real time clock
+
+    
+    identifier: 0x1a8b1a28
+    extends: _sensor
```

Comment:
we should also have precision register, in parts per million, u16.16 (this is not error though)

Reply:
wrong service for this comment?

Reply:
no, for this one; DS3231 is 2ppm; crystals are often ~10ppm depending on temp

Reply:
we have error, resolution as system registers, precision added?

Reply:
in some sense the "precision" here is "rate of error"; it typically doesn't make sense to state it, but here it accumulates with time; I wouldn't add anything to system just yet

Reply:
I just learned there are atomic clocks (microchip sells one for $5k; it's only a few cm in size) that have precision of 0.00005 ppm. So I guess precision as u16.16 ppm would work - this from 0.00002 ppm to 6% (I don't think you could call something with 6% precision a "real time" clock).